internal/hash/file_hasher.go

```diff
@@ -0,0 +1,75 @@
+package hash
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"time"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/cespare/xxhash"
```

Comment:
xxhash is a fast hash algorithm. For more info see here: https://xxhash.com/
<sub>this link is safe for work</sub>